src/python/grpcio/tests/_runner.py

```diff
@@ -43,13 +44,6 @@
 from tests import _loader
 from tests import _result
 
-# This number needs to be large enough to outpace output on stdout and stderr
-# from the gRPC core, otherwise we could end up in a potential deadlock. This
-# stems from the OS waiting on someone to clear a filled pipe buffer while the
-# GIL is held from a write to stderr from gRPC core, but said someone is in
-# Python code thus necessitating GIL acquisition.
-_READ_BYTES = 2**20
-
 
 class CapturePipe(object):
   """A context-manager pipe to redirect output to a byte array.
```

Comment:
How about removing no-longer-existent class-private attributes from this class-public doc string?

Any need to update the summary line?